pkg/controller/core.oam.dev/v1alpha2/application/apply.go

```diff
@@ -124,13 +113,12 @@ func (h *appHandler) apply(ctx context.Context, appRev *v1beta1.ApplicationRevis
 		h.setInplace(false)
 	}
 
-	// don't create components and AC if revision-only annotation is set
+	// don't create components if revision-only annotation is set
```

Comment:
do we still have components after the refactor?

Reply:
I would like to do refactor component once refactoring app is done. Because that will involve a huge number of changes in appfile pkg. This PR or branch cannot hold more (T.T)